libraries/protocol/include/graphene/protocol/fee_schedule.hpp

```diff
@@ -37,116 +37,74 @@ namespace graphene { namespace protocol {
    template<typename Operation>
    class fee_helper {
      public:
-      const typename Operation::fee_parameters_type& cget(const fee_parameters::flat_set_type& parameters)const
+      const typename Operation::fee_parameters_type& get(const fee_parameters::flat_set_type& parameters)const
       {
          auto itr = parameters.find( typename Operation::fee_parameters_type() );
-         FC_ASSERT( itr != parameters.end() );
-         return itr->template get<typename Operation::fee_parameters_type>();
-      }
-   };
+         if( itr != parameters.end() )
+            return itr->template get<typename Operation::fee_parameters_type>();
 
-   template<>
-   class fee_helper<account_create_operation> {
-     public:
-      const account_create_operation::fee_parameters_type& cget(const fee_parameters::flat_set_type& parameters)const
-      {
-         auto itr = parameters.find( account_create_operation::fee_parameters_type() );
-         FC_ASSERT( itr != parameters.end() );
-         return itr->get<account_create_operation::fee_parameters_type>();
+         static typename Operation::fee_parameters_type dummy;
+         return dummy;
       }
-      typename account_create_operation::fee_parameters_type& get(fee_parameters::flat_set_type& parameters)const
+      typename Operation::fee_parameters_type& mutable_get(fee_parameters::flat_set_type& parameters)const
```

Comment:
Only `account_create_operation` requires a `mutable_get` function. What are the pros and cons to expand it to a template?

Actually, the "account_fee_scale_bitshifts" was always `0` on BitShares mainnet, which means we've never used that fee bit-shifting feature. I doubt its usefulness. BTW IIRC the feature caused unexpected troubles on the SEER blockchain/project (seer.best) when the team tried to massively on-board new users.

Reply:
Backed out changes to fee_schedule